todo_app/app.py

```diff
@@ -3,40 +3,221 @@
 
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
-from todo_app.data.session_items import get_item
-from todo_app.data.session_items import save_item
+from todo_app.data.session_items import get_items, add_item, get_item, save_item
+from todo_app.data.Items import ToDo
+import requests
+import json
+import os
+import logging
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
-def index():
-   
-	return render_template("index.html",items = get_items())
+toDoId = os.environ['TOID']
+doingId = os.environ['DOINGID']
+doneId = os.environ['DONE']
```

Comment:
The README needs to mention these as well.

And you still have a few places below where you have hardcoded e.g. `'idList': '6005828032dafa5707bf5dc5'` instead of using these variables.